quetz_frontend/backend.py

```diff
@@ -117,23 +117,25 @@ def index(
         )
 
         path = frontend_dir / "static" / file_name
-        if path.exists() and under_frontend_dir(frontend_dir, path):
+        if path.exists():
```

Comment:
Are you sure it is a good idea to remove the security check `under_frontend_dir(frontend_dir, path)`?
The reason for this check is not to allow requesting files that are not under the `frontend_dir` directory.

Reply:
sorry for merging early @hbcarlos

Reply:
It will always be true as `path = frontend_dir / "static" / file_name` (see line above L119)

Reply:
Great! Thank you soo much @fcollonval!